Automattic/wp-calypso
# Stats: Adding date picker events

Related to #83498

## Proposed Changes

* add date picker tracking events to monitor and improve usage

## Testing Instructions



* go to the live branch
* ensure date picker and shortcuts work as expected

## Pre-merge Checklist
